src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerTest.java

```diff
@@ -60,6 +60,27 @@ public void shouldImportRealm() {
         }
     }
 
+    @Test
+    public void shouldImportMultipleRealms() {
+        try (KeycloakContainer keycloak = new KeycloakContainer().
+            withRealmImportFile(TEST_REALM_JSON).
+            withRealmImportFile("another-realm.json")) {
```

Comment:
While this is a great possibility to add multiple realm files, WDYT about an additional `.withRealmImportFiles(String... files)` method, to import multiple files with one command? I know it's "just" conveniance - but conveniance! 😉

Reply:
sure, method added.

Reply:
great, thanks